rocketpool-cli/commands/service/config.go

```diff
@@ -131,8 +131,14 @@ func configureService(c *cli.Context) error {
 				suffix := config.GetContainerName(container)
 				name := cfg.GetDockerArtifactName(suffix)
 				fmt.Printf("Stopping %s... ", name)
-				rp.StopContainer(name)
-				fmt.Print("done!\n")
+				err := rp.StopContainer(name)
+				if err != nil {
+					fmt.Print("error!")
+					fmt.Fprintf(os.Stderr, "Error stopping container %s: %s\n", name, err.Error())
```

Comment:
Is this going to end up printing to the terminal on the same line as the above since there's not a newline first?

Reply:
if stdout and stderr are the same file, yeah. we'll have to hoist the err message outside the loop, or move the blank println back in